razorpay-payments.php

```diff
@@ -222,7 +222,7 @@ protected function verifyOrderAmount($razorpayOrderId, $orderId)
 
             $razorpayOrderArgs = array(
                 'id'        => $razorpayOrderId,
-                'amount'    => (int) $order->order_total*100,
+                'amount'    => $order->order_total*100,
```

Comment:
Dropping the int also means we may be passing float to our systems lower down which may cause issues, no? Did some tests:

```>>> $a  = 159.2;
=> 159.2
>>> (int) $a * 100;
=> 15900
>>> (int) ($a * 100);
=> 15919
>>> 159.2 * 100
=> 15920.0
>>> (int)(159.2 * 100);
=> 15919
>>> (159.2 * 100);
=> 15920.0
>>> floor($a * 100);
=> 15919.0
>>> round($a * 100);
=> 15920.0
>>> round($a * 100);
=> 15920.0
>>> intval($a * 100);
=> 15919
>>> ceil($a * 100);
=> 15920.0
>>> round($a * 100);
=> 15920.0
>>> (int) round ($a*100);
=> 15920
```

The typecast + round approach looks cleaner to me. /cc @73sl4